msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -14,7 +14,7 @@ The resource-specific consent (RSC) permissions model, originally developed for
 
 > [!NOTE]
 >
-> The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md).
+> The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md) and will only be enabled after a new installation into a chat.
```

Comment:
Also clarify that the bot continues to receive all messages even when the client switches out of preview.

Reply:
Clarified this behavior in the notes.